bin/rails

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+APP_PATH = File.expand_path('../config/application', __dir__)
+require_relative '../config/boot'
+require 'rails/commands'
```

Comment:
End of line missing

Reply:
Gah! Thanks for the spot. Fixed in https://github.com/alphagov/calculators/pull/199/commits/1d1a1b71cf075b6f38a9cabd13e18386bcc33f66